docs/README.md

```diff
@@ -0,0 +1,69 @@
+# Kite for Visual Studio Code
+
+This extensions adds support for Kite in Visual Studio Code, including:
+
+- Completion Lists
+- Signature Help
+- Quick Info
+- Goto Definition 
+
+## How to use this extension
+
+This extension is automatically installed by Kite when you select Visual Studio Code in the list of plugins.
+
+### Startup
+
+When starting VSCode with Kite's plugin for the first time, the Kite's tour will be displayed in the active pane.
+
+![Kite Tour](./images/kite-tour.png)
+
+This tour will only be displayed once, if you want to see it again on next startup you can activate the `kite.showTourOnStartup` setting.
+
+### Status Bar
+
+The Kite icon in the status bar displays the state of Kite for the current file. By clicking on the icon you open the status panel with additional information.
+
+![kite status bar](./images/kite-status-bar.png)
+
+The icon in the status bar can take three different colors:
+
+- blue: The Kite Engine is available and functioning properly.<br/>![kite status ready](./images/kite-status-ready.png)
+- gray: There's either no open file or, if there's an active file, the file is either not supported or not whitelisted.<br/>![kite status not whitelisted](./images/kite-status-non-whitelisted.png)
+- red: Something went wrong when the plugin tried to contact the Kite service on your computer. Depending on the issue, the status panel can offer actions to solve the problem.<br/>![kite status not running](./images/kite-status-not-running.png)
+
+### Quick Info
+
+![kite hover](./images/kite-hover.png)
+
+You can see up to three links in the popup:
+
+- `def` will open the file where this symbol is defined
```

Comment:
change this to:
- `def` will open the file where this symbol is defined (this may not be available if Kite cannot find the definition)